app/pages/exportImport/PerformImport.tsx

```diff
@@ -228,16 +331,24 @@ export default function PerformImport({ location }: Props) {
                         </Columns.Column>
                         <Columns.Column>
                             <div className={styles.importedList}>
-                                <div className="flexChildFill flexColumn justifyCenter">
+                                <div className={styles.importedListInner}>
                                     {importedData.identities.map(
                                         (identity: Identity) => (
                                             <div
                                                 key={identity.id}
                                                 className={styles.importSection}
                                             >
-                                                <h3>
+                                                <h3 className="mB0">
                                                     <b>ID:</b> {identity.name}
                                                 </h3>
+                                                <p className="mT0 body3 bodyLight">
+                                                    {messages[identity.id] &&
+                                                        `(${
+                                                            messages[
+                                                                identity.id
+                                                            ]
+                                                        })`}
+                                                </p>
```

Comment:
I think this should be on the same line as with account conflict messages for consistency.

Reply:
I've moved it to the same line + added `textFaded` and removed `body3`.

Reply:
I think you forgot the part where you wanted to remove `body3`? 😉